package.json

```diff
@@ -102,24 +102,35 @@
     ],
     "rules": {
       "angular/angularelement": "error",
-      "angular/di": "error",
-      "angular/di-order": "error",
+      "angular/di": "warn",
+      "angular/di-order": "warn",
       "angular/di-unused": "error",
       "angular/document-service": "error",
       "angular/log": "error",
       "angular/module-dependency-order": "error",
       "angular/no-service-method": "error",
-      "angular/no-services": "error",
+      "angular/no-services": "warn",
       "angular/window-service": "error",
       "compat/compat": "error",
       "arrow-parens": "off",
       "constructor-super": "off",
       "generator-star-spacing": "off",
+      "max-len": [
+        "error",
+        {
+          "code": 120,
```

Comment:
I'd favor not making an exception around line length. descriptive function and variable names are good -- and refactoring to better abstractions when 80 characters feels restrictive is good too.

I agree it's a question of style. The meta-value in not making exceptions about style is high. The first rule of code-style-club is don't talk about code-style-club. [`-0.9`](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions).